lib/assets/javascripts/turbograft/initializers.coffee

```diff
@@ -50,9 +50,8 @@ TurboGraft.handlers.remoteFormHandler = (ev) ->
 documentListenerForButtons = (eventType, handler, useCapture = false) ->
   document.addEventListener eventType, (ev) ->
     target = ev.target
-    return if !target
 
-    while target != document && (typeof target != "undefined")
+    while target != document && (typeof target != "undefined") && (target != null)
```

Comment:
Just for the future, `typeof undefined == 'undefined'` and `typeof something_never_declared == 'undefined'`